homeassistant/components/totalconnect/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: todo
+  test-before-setup: todo
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Total Connect entities do not subscribe to events.
+  dependency-transparency: done
```

Comment:
I am missing a CI pipeline that handles the publishing of the dependency. We require this for increased transparency

Reply:
I will have to figure this out.  set to `todo`